ui/components/extensions/adapters/adapters.js

```diff
@@ -58,19 +58,22 @@ const Adapters = ({ updateProgress, classes }) => {
   const handleAdapterDeployment = (payload, msg, selectedAdapter, adapterId) => {
     updateProgress({ showProgress: true });
 
+    const isEnableAction = payload.status === 'ENABLED';
```

Comment:
Define (if it does not exist already) a constant inside constants.js and use that here as well on line 74 (you would have to use toLowerCase() or toUpperCase() depending on what you define the value for that constant)